plugins/single_plugins/grid.cpp

```diff
@@ -255,24 +261,69 @@ class wayfire_grid : public wf::plugin_interface_t
             output->add_activator(keys[i], &bindings[i]);
         }
 
+        restore = [=] (wf_activator_source, uint32_t)
+        {
+            auto view = output->get_active_view();
+            restore_view(view);
+        };
+
+        restore_opt = section->get_option("restore", "last");
+        restore_opt_str = restore_opt->as_string();
+        restore_opt_changed = [=] ()
+        {
+            if (restore_opt_str != "last")
```

Comment:
IIRC you don't need to check, if the binding isn't found, `rem_binding` is a no-op.
And then you also don't need `restore_opt_str` at all.